### PR TITLE
Minor nits: pootle_fs and others

### PR DESCRIPTION
**Pull request by unho**
- Docs changes
- Typo
- Add requirement for pootle_fs_git
- Moved `POOTLE_FS_WORKING_PATH`

### CODE REVIEW

File: requirements/_pootle_fs.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# pootle_fs requirements
+
+# If you want to use pootle_fs git plugin
+-e git+https://github.com/translate/pootle_fs_git@pootle_fs_cleanups#egg=pootle-fs-git
```
Comment (unho) [inline requirements/_pootle_fs.txt]: Maybe this should be disabled by default.

Comment (unho) [inline requirements/_pootle_fs.txt]: Changed.

File: pootle/.pootle_fs/tmp/README [hunk 2]
```diff
@@ -1 +1,3 @@
 This directory (pootle_fs) is the working directory for pootle_fs.
+
+NEVER touch anything on this directory.
```
Comment (phlax) [inline pootle/.pootle_fs/tmp/README]: i think we neen some more information here

this dir has the potential to get very large, so we need to communicate that they might want/need to place it somewhere specific and make the necessary adjustments to their settings

also rem that this folders ends up in an egg dir in the middle of nowhere in a production env, so im thinking there needs to be docs elsewhere explaining all of this

Comment (unho) [inline pootle/.pootle_fs/tmp/README]: Will adjust to highlight that both in docs and here.

**Comment by phlax** [comment]
other than the docs changes, lgtm

**Pull request merged**